docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -572,3 +573,52 @@ test('Fixture2Test3', () => {});
 
 // Only tests in Fixture1 and the Fixture2Test2 test are run
 ```
+
+## Inject Scripts into Tested Pages
+
+TestCafe allows you to [inject custom scripts](../using-testcafe/common-concepts/inject-scripts-into-tested-pages.md) into pages visited during the tests. You can add scripts that mock browser API or provide helper functions.
```

Comment:
One note:
The injected script is executed in the `head` tag. At that moment, the page body not yet fully parsed and DOM is not completely calculated.
It means you cannot perform DOM manipulation at that moment.
For example, the `document.body.style.backgroundColor = 'green';` will raise an error because at that moment `document.body` is not present. 
If you want to execute in client script the code that performs DOM manipulations then you should wrap you code in 'DOMContentLoaded' event listener;
```js
const scriptContent = `
window.addEventListener('DOMContentLoaded', function () {
   document.body.style.backgroundColor = 'green'
});
`

.clientScripts({ content: scriptContent})
```
You need to wait